gpt4all/readme.md

```diff
@@ -26,7 +26,7 @@ point to navigate and move around files, directories, etc.
 
 To start, clone this repository:
 ```commandline
-git clone https://github.com/opentensor/gpt4all_tuning 
+git clone https://github.com/salahawk/bittensor-model-finetune/gpt4all 

```

Comment:
should be `https://github.com/salahawk/bittensor-model-finetune`?
don't need `gpt4all` appended?